jupyter/tensorflow/ubi9-python-3.12/Dockerfile.cuda

```diff
@@ -189,6 +189,11 @@ RUN ACCEPT_EULA=Y dnf install -y mssql-tools18 unixODBC-devel && dnf clean all &
 
 ENV PATH="$PATH:/opt/mssql-tools18/bin"
 
+# hdf5 is needed for h5py
+RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm && \
+    dnf install -y hdf5-devel && \
+    dnf clean all
```

Comment:
@coderabbitai mixing centos and ubi is bad practice, but using epel should be just fine, although it will be extra work with AIPCC to find suitable build internally

> - Jiri Danek found a reference on [developers.redhat.com/articles/ubi-faq](http://developers.redhat.com/articles/ubi-faq) stating 'don’t add CentOS packages (because they will remove supportability). Adding CentOS packages turns the image into a Frankenstein.'

reference for not using ubi + centos https://redhat-internal.slack.com/archives/C06B3AVCJ64/p1752865151332839?thread_ts=1720079270.728929&cid=C06B3AVCJ64